tests/test_table.py

```diff
@@ -59,4 +59,4 @@ def test_wb_table_final_data(self):
             #    not 'word_distance' or 'hinge_loss'
             for i,data_row in enumerate(experiment.wb_table.final_table.data):
                 assert data_row[:6] == test_row_list[i][:6]
-                assert data_row[-1] == test_row_list[i][-1]
```

Comment:
why remove this assertion?

Reply:
It's present on my local version, not sure what happened.  I'll re-push.